example/.eslintrc

```diff
@@ -0,0 +1,16 @@
+{
+  "plugins": ["json"],
+  "rules": {
+      "json/*": ["warn"],
+      "json/duplicate-key": "error",
+      "json/trailing-comma": "error"
+    },
+    "overrides": [
+        {
+            "files": ["samples/jsonc.json"],
+            "rules": {
+                "json/json": ["warn", {"allowComments": true}]
```

Comment:
It can be specificaly set with `json/comment-not-permitted`.

By default, `json/json` (and it's alias `json/*`), activate this rules (as comments are not part of the json spec). the option `allowComments` allow user to activate everything but this rule.

But it's strange you had to made the change since `json/json` and `json/*` are [supposed to be the same](https://github.com/azeemba/eslint-plugin-json/pull/26/files#diff-6d186b954a58d5bb740f73d84fe39073R121-R122)
I'm going to try to replicate